test/system/irohad_test.cpp

```diff
@@ -36,29 +38,51 @@ using namespace std::chrono_literals;
 using namespace common_constants;
 using iroha::operator|;
 
+static logger::LoggerManagerTreePtr getIrohadTestLoggerManager() {
+  static logger::LoggerManagerTreePtr irohad_test_logger_manager;
+  if (!irohad_test_logger_manager) {
+    irohad_test_logger_manager =
+        std::make_shared<logger::LoggerManagerTree>(logger::LoggerConfig{
+            logger::LogLevel::kInfo, logger::kDefaultLogPatterns});
+  }
+  return irohad_test_logger_manager->getChild("IrohadTest");
+}
+
 class IrohadTest : public AcceptanceFixture {
  public:
-  IrohadTest() : kAddress("127.0.0.1"), kPort(50051) {}
+  IrohadTest()
+      : kAddress("127.0.0.1"),
+        kPort(50051),
```

Comment:
Can we use our port manager? Can we avoid hard coding the port?

Reply:
Maybe we can, but it's not related to the new logger integration, and actually we do not need it here now.